languages/csharp/AppEncryption/README.md

```diff
@@ -197,24 +71,21 @@ AWSConfigs.AWSRegion = "us-west-2";
 
 // Build the DynamoDB Metastore.
 IMetastorePersistence<JObject> dynamoDbMetastorePersistence = DynamoDbMetastorePersistenceImpl.NewBuilder().Build();
+```
 
-// Use the Metastore for the session factory
-using (AppEncryptionSessionFactory appEncryptionSessionFactory = AppEncryptionSessionFactory.NewBuilder("productId", "systemId")
-     .WithMetastorePersistence(dynamoDbMetastorePersistence)
-     .WithCryptoPolicy(policy)
-     .WithKeyManagementService(keyManagementService)
-     .Build()) {
+#### In-memory Metastore (FOR TESTING ONLY)
 
-    // ...
-}
+```c#
+IMetastorePersistence<JObject> metastorePersistence = metastorePersistence = new MemoryPersistenceImpl<JObject>();
```

Comment:
previous syntax errors fixed but looks like we have an extra `metastorePersistence =`